.github/workflows/firesim-run-tests.yml

```diff
@@ -82,6 +82,9 @@ jobs:
         run: ./.github/scripts/launch-manager-instance.py
       - name: Setup N Github Actions Runners on AWS instance
         run: ./.github/scripts/setup-manager-self-hosted.py
+      - name: Catch potentially orphaned manager
+        if: ${{ failure() || cancelled() }}
```

Comment:
Does this only catch failures/cancellations that happen in the "run" line above? Don't you want this after the "launch-manager-instance.py" script run as well?

Reply:
Resolved offline. This catches all prior steps.